google/cloud/pubsub/config-mocks.cmake.in

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(CMakeFindDependencyMacro)
+find_dependency(google_cloud_cpp_pubsub)
+find_dependency(gtest)
```

Comment:
There is a bit of a mess with `FindGTest` (the built-in module in CMake) vs. the config file installed by googletest:

https://cmake.org/cmake/help/latest/module/FindGTest.html

Maybe we should install a file like this and include it instead?

https://github.com/googleapis/google-cloud-cpp/blob/main/cmake/FindGMockWithTargets.cmake

Reply:
As you pointed out, there is incremental progress to be made by installing in cases where `FindGTest` works. We can figure out how to test the other case in our CI later.